<!DOCTYPE html>
<html lang="en">
<html>
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link rel="stylesheet" href="styles/normalize.css">
  <link rel="stylesheet" href="styles/style.css">
  <link rel="icon" type="image" href="https://code.s3.yandex.net/web-code/rules-favicon.ico">
  <title>Journal of Everything</title>    
</head>
<body>
  <header class="header">
    <div class="overlay">
      <h1 class="header-title">Journal of Everything<br>
	  <i>peer-reviewed</i></h1>
    </div>
  </header>
  <nav class='menu'>
    <a href='#' class='menu-link'>Choose topic</a>
    <a href='#' class='menu-link'>Top New</a>
    <a href='#' class='menu-link'>Enter account</a>
  </nav>
  <main class="content flex-row">
	<!--Card 1-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://pictures.s3.yandex.net/frontend-developer/free-course/card-1.jpg">
		<p class="card-title">TIRR: a potential front runner in HDR race − hypotheses and perspectives</p>
		<div class="card-author">
			<p>A. A. Anuchina, A. V. Lavrov & S. A. Smirnikhina</p>
		</div>
		<div class="card-author-aff">
			<p>Research Center for Medical Genetics</p>
		</div>
		<p class="card-text">The majority of CRISPR-Cas9 methods for mutations correction are oriented on gene editing through homologous recombination that is normally restrained by non-homologous end joining (NHEJ). A recently identified protein TIRR can bind a 53BP1 protein, a key effector of NHEJ, and inhibit its recruitment to double-strand break loci. Several studies elucidated the molecular mechanisms of TIRR-53BP1 binding and established bidirectional role of TIRR in 53BP1 functions and stability. It was proved that overexpression of TIRR promotes the double-strand break repair through homologous recombination. All findings, which were described in the review, allow assuming TIRR as a suitable target for enhancing efficacy of genome editing through homology directed repair.</p>
		
    </article>
    <!--Card 2-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://pictures.s3.yandex.net/frontend-developer/free-course/card-2.jpg">
		<p class="card-title">Ways of improving precise knock-in by genome-editing technologies</p>
		<div class="card-author">
			<p>Svetlana A Smirnikhina 1, Arina A Anuchina 2, Alexander V Lavrov 2 3</p>
		</div>
		<div class="card-author-aff">
			<p>1Laboratory of Mutagenesis, Federal State Budgetary Institution "Research Centre for Medical Genetics", Moskvorechie, 1, Moscow, 115522, Russia. smirnikhinas@gmail.com.
2Laboratory of Mutagenesis, Federal State Budgetary Institution "Research Centre for Medical Genetics", Moskvorechie, 1, Moscow, 115522, Russia.
3Department of Molecular and Cellular Genetics, The Russian National Research Medical University Named after N.I. Pirogov, Moscow, Russia.</p>
		</div>		
		<p class="card-text">Despite the recent discover of genome-editing methods, today we can say these approaches have firmly entered our life. Two approaches-knocking out malfunctioning gene allele or correcting the mutation with precise knock-in-can be used in hereditary monogenic diseases treatment. The latter approach is relatively ineffective. Modern data about the ways of repair of double-strand DNA breaks formed by nucleases are presented in this review. The main part of the review is devoted to the ways of increasing precise and effective knock-in: inhibition of non-homologous end joining and stimulation of homology-directed repair key factors, use of small molecules with unknown mechanism of action, cell-cycle synchronization and cell-cycle-dependent activity of Cas9, donor molecule design, selection, alternative methods for insertion and other approaches.</p>
    </article>
    <!--Card 3-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs12920-020-00735-8/MediaObjects/12920_2020_735_Fig3_HTML.png">
		<p class="card-title">Genome scale analysis of pathogenic variants targetable for single base editing</p>
		<div class="card-author">
			<p>Alexander V Lavrov 1, Georgi G Varenikov 2, Mikhail Yu Skoblov 3 2 4</p>
		</div>
		<div class="card-author-aff">
			<p>1Research Center for Medical Genetics, Moscow, Russia. alexandervlavrov@gmail.com.
2Moscow Institute of Physics and Technology, Dolgoprudny, Russia.
3Research Center for Medical Genetics, Moscow, Russia.
4School of Biomedicine, Far Eastern Federal University, Vladivostok, Russia.</p>
		</div>
		<p class="card-text">Background: Single nucleotide variants account for approximately 90% of all known pathogenic variants responsible for human diseases. Recently discovered CRISPR/Cas9 base editors can correct individual nucleotides without cutting DNA and inducing double-stranded breaks. We aimed to find all possible pathogenic variants which can be efficiently targeted by any of the currently described base editors and to present them for further selection and development of targeted therapies.

Methods: ClinVar database (GRCh37_clinvar_20171203) was used to search and select mutations available for current single-base editing systems. We included only pathogenic and likely pathogenic variants for further analysis. For every potentially editable mutation we checked the presence of PAM. If a PAM was found, we analyzed the sequence to find possibility to edit only one nucleotide without changing neighboring nucleotides. The code of the script to search Clinvar database and to analyze the sequences was written in R and is available in the appendix.

Results: We analyzed 21 editing system currently reported in 9 publications. Every system has different working characteristics such as the editing window and PAM sequence. C > T base editors can precisely target 3196 mutations (46% of all pathogenic T > C variants), and A > G editors - 6900 mutations (34% of all pathogenic G > A variants).

Conclusions: Protein engineering helps to develop new enzymes with a narrower window of base editors as well as using new Cas9 enzymes with different PAM sequences. But, even now the list of mutations which can be targeted with currently available systems is huge enough to choose and develop new targeted therapies.</p>
		
    </article>
    <!--Card 4-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/0c134010-5183-4f6a-8a39-c26275f37649/gr6.jpg">
		<p class="card-title">Global Public Perceptions of Genomic Data Sharing: What Shapes the Willingness to Donate DNA and Health Data?</p>
		<div class="card-author">
			<p>Anna Middleton, Richard Milne, Mohamed A.Almarri, Shamim Anwer, Jerome Atutornu, Elena E.Baranova, Paul Bevan, Maria Cerezo, Yali Cong, Christine Critchley, Josepine Fernow, Peter Goodhand, Qurratulain Hasan, Aiko Hibino, Gry Houeland, Heidi C.Howard, S. Zakir Hussain, Charlotta Ingvoldstad Malmgren, Vera .Izhevskaya, Aleksandra Jędrzejak, Cao Jinhong, Megumi Kimura, Erika Kleiderman, Brandi Leach, Keying Liu, Deborah Mascalzoni, Álvaro Mendes, Jusaku Minari, Nan Wang, Dianne Nicol, Emilia Niemiec, Christine Patch, Jack Pollard, Barbara Prainsack, Marie Rivière, Lauren Robarts, Jonathan Roberts, Virginia Romano, Haytham A.Sheerah, James Smith, Alexandra Soulier, Claire Steed, Vigdís Stefànsdóttir, Cornelia Tandre, Adrian Thorogood, Torsten H.Voigt, Anne V.West, Go Yoshizawa, Katherine I.Morley.3</p>
		</div>
		<div class="card-author-aff">
			<p>Middleton et al., Global Public Perceptions of Genomic Data Sharing: What Shapes the Willingness to Donate DNA and Health Data? The American Journal of Human Genetics (2020).</p>
		</div>
		<p class="card-text">Analyzing genomic data across populations is central to understanding the role of genetic factors in health and disease. Successful data sharing relies on public support, which requires attention to whether people around the world are willing to donate their data that are then subsequently shared with others for research. However, studies of such public perceptions are geographically limited and do not enable comparison. This paper presents results from a very large public survey on attitudes toward genomic data sharing. Data from 36,268 individuals across 22 countries (gathered in 15 languages) are presented. In general, publics across the world do not appear to be aware of, nor familiar with, the concepts of DNA, genetics, and genomics. Willingness to donate one’s DNA and health data for research is relatively low, and trust in the process of data’s being shared with multiple users (e.g., doctors, researchers, governments) is also low. Participants were most willing to donate DNA or health information for research when the recipient was specified as a medical doctor and least willing to donate when the recipient was a for-profit researcher. Those who were familiar with genetics and who were trusting of the users asking for data were more likely to be willing to donate. However, less than half of participants trusted more than one potential user of data, although this varied across countries. Genetic information was not uniformly seen as different from other forms of health information, but there was an association between seeing genetic information as special in some way compared to other health data and increased willingness to donate. The global perspective provided by our “Your DNA, Your Say” study is valuable for informing the development of international policy and practice for sharing genomic data. It highlights that the research community not only needs to be worthy of trust by the public, but also urgent steps need to be taken to authentically communicate why genomic research is necessary and how data donation, and subsequent sharing, is integral to this.</p>
		
    </article>
	  <!--Card 5-->
	  <article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/0c134010-5183-4f6a-8a39-c26275f37649/gr6.jpg">
		<p class="card-title">Global Public Perceptions of Genomic Data Sharing: What Shapes the Willingness to Donate DNA and Health Data?</p>
		<div class="card-author">
			<p>Anna Middleton, Richard Milne, Mohamed A.Almarri, Shamim Anwer, Jerome Atutornu, Elena E.Baranova, Paul Bevan, Maria Cerezo, Yali Cong, Christine Critchley, Josepine Fernow, Peter Goodhand, Qurratulain Hasan, Aiko Hibino, Gry Houeland, Heidi C.Howard, S. Zakir Hussain, Charlotta Ingvoldstad Malmgren, Vera .Izhevskaya, Aleksandra Jędrzejak, Cao Jinhong, Megumi Kimura, Erika Kleiderman, Brandi Leach, Keying Liu, Deborah Mascalzoni, Álvaro Mendes, Jusaku Minari, Nan Wang, Dianne Nicol, Emilia Niemiec, Christine Patch, Jack Pollard, Barbara Prainsack, Marie Rivière, Lauren Robarts, Jonathan Roberts, Virginia Romano, Haytham A.Sheerah, James Smith, Alexandra Soulier, Claire Steed, Vigdís Stefànsdóttir, Cornelia Tandre, Adrian Thorogood, Torsten H.Voigt, Anne V.West, Go Yoshizawa, Katherine I.Morley.3</p>
		</div>
		<div class="card-author-aff">
			<p>Middleton et al., Global Public Perceptions of Genomic Data Sharing: What Shapes the Willingness to Donate DNA and Health Data? The American Journal of Human Genetics (2020).</p>
		</div>
		<p class="card-text">Analyzing genomic data across populations is central to understanding the role of genetic factors in health and disease. Successful data sharing relies on public support, which requires attention to whether people around the world are willing to donate their data that are then subsequently shared with others for research. However, studies of such public perceptions are geographically limited and do not enable comparison. This paper presents results from a very large public survey on attitudes toward genomic data sharing. Data from 36,268 individuals across 22 countries (gathered in 15 languages) are presented. In general, publics across the world do not appear to be aware of, nor familiar with, the concepts of DNA, genetics, and genomics. Willingness to donate one’s DNA and health data for research is relatively low, and trust in the process of data’s being shared with multiple users (e.g., doctors, researchers, governments) is also low. Participants were most willing to donate DNA or health information for research when the recipient was specified as a medical doctor and least willing to donate when the recipient was a for-profit researcher. Those who were familiar with genetics and who were trusting of the users asking for data were more likely to be willing to donate. However, less than half of participants trusted more than one potential user of data, although this varied across countries. Genetic information was not uniformly seen as different from other forms of health information, but there was an association between seeing genetic information as special in some way compared to other health data and increased willingness to donate. The global perspective provided by our “Your DNA, Your Say” study is valuable for informing the development of international policy and practice for sharing genomic data. It highlights that the research community not only needs to be worthy of trust by the public, but also urgent steps need to be taken to authentically communicate why genomic research is necessary and how data donation, and subsequent sharing, is integral to this.</p>
		
    </article>
	<!--Card 6-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://ars.els-cdn.com/content/image/1-s2.0-S1873506120302348-gr1.jpg">
		<p class="card-title">Generation of induced pluripotent stem cell line (RCMGi001-A) from human skin fibroblasts of a cystic fibrosis patient with p.F508del mutation</p>
		<div class="card-author">
			<p>Kondrateva E, Adilgereeva E, Amelina E, Tabakov V, Demchenko A, Ustinov K, Yasinovsky M, Voronina E, Lavrov A, Smirnikhina S.</p>
		</div>
		<div class="card-author-aff">
			<p>a
Research Centre for Medical Genetics, Moscow 115522, Russian Federation
b
The Research Institute of Pulmonology, Moscow 115682, Russian Federation</p>
		</div>
		<p class="card-text">Skin fibroblasts obtained from a 27-year-old man with clinically manifested and genetically proven (F508del/F508del) cystic fibrosis were successfully transformed into induced pluripotent stem cells (iPSCs) by using Sendai virus-based reprogramming vectors including the four Yamanaka factors, OCT3/4, SOX2, KLF4, and c-MYC. The iPSCs showed a normal karyotype, expressed pluripotency markers and exhibited the potential to differentiate into three germ layers in spontaneous differentiation assay. This iPSC line may be subsequently used for development of a personalized etiotropic treatment including genome editing, and for disease modelling and drug screening.</p>
		
    </article>
	<!--Card 7-->
	<article class="card">
		<div>
			<p class="card-article-type">Review</p>
			<p class="card-article-date">Published: 08 February 2020</p>
		</div>
		<img class="card-image" src="https://www.frontiersin.org/files/Articles/538141/fcell-08-00386-HTML/image_m/fcell-08-00386-g006.jpg">
		<p class="card-title">1Q12 Loci Movement in the Interphase Nucleus Under the Action of ROS Is an Important Component of the Mechanism That Determines Copy Number Variation of Satellite III (1q12) in Health and Schizophrenia</p>
		<div class="card-author">
			<p>Marina Sergeevna Konkova, Elizaveta Sergeevna Ershova, Ekaterina Alekseevna Savinova, Elena Mikhailovna Malinovskaya, Galina Vasilievna Shmarina, Andrey Vladimirovich Martynov, Roman Vladimirovich Veiko, Nataly Vyacheslavovna Zakharova, Pavel Umriukhin, Georgy Petrovich Kostyuk, Vera Leonidovna Izhevskaya, Sergey Ivanovich Kutsev, Natalia Nikolaevna Veiko and Svetlana Victorovna Kostyuk.</p>
		</div>
		<div class="card-author-aff">
			<p>1Federal State Budgetary Scientific Institution, Research Centre for Medical Genetics, Moscow, Russia
2N.A. Alekseev Clinical Psychiatric Hospital No 1, Moscow Healthcare Department, Moscow, Russia
3I.M. Sechenov First Moscow State Medical University of the Ministry of Health of the Russian Federation, Moscow, Russia
4P.K. Anokhin Institute of Normal Physiology, Moscow, Russia</p>
		</div>
		<p class="card-text">Introduction: Genome repeat cluster sizes can affect the chromatin spatial configuration and function. Low-dose ionizing radiation (IR) induces an adaptive response (AR) in human cells. AR includes the change in chromatin spatial configuration that is necessary to change the expression profile of the genome in response to stress. The 1q12 heterochromatin loci movement from the periphery to the center of the nucleus is a marker of the chromatin configuration change. We hypothesized that a large 1q12 domain could affect chromatin movement, thereby inhibiting the AR.

Materials and Methods: 2D fluorescent in situ hybridization (FISH) method was used for the satellite III fragment from the 1q12 region (f-SatIII) localization analysis in the interphase nuclei of healthy control (HC) lymphocytes, schizophrenia (SZ) patients, and in cultured mesenchymal stem cells (MSCs). The localization of the nucleolus was analyzed by the nucleolus Ag staining. The non-radioactive quantitative hybridization (NQH) technique was used for the f-SatIII fragment content in DNA analysis. Satellite III fragments transcription was analyzed by reverse transcriptase quantitative PCR (RT-qPCR).

Results: Low-dose IR induces the small-area 1q12 domains movement from the periphery to the central regions of the nucleus in HC lymphocytes and MSCs. Simultaneously, nucleolus moves from the nucleus center toward the nuclear envelope. The nucleolus in that period increases. The distance between the 1q12 domain and the nucleolus in irradiated cells is significantly reduced. The large-area 1q12 domains do not move in response to stress. During prolonged cultivation, the irradiated cells with a large f-SatIII amount die, and the population is enriched with the cells with low f-SatIII content. IR induces satellite III transcription in HC lymphocytes. Intact SZ patients’ lymphocytes have the same signs of nuclei activation as irradiated HC cells.

Conclusion: When a cell population responds to stress, cells are selected according to the size of the 1q12 domain (the f-SatIII content). The low content of the f-SatIII repeat in SZ patients may be a consequence of the chronic oxidative stress and of a large copies number of the ribosomal repeats.</p>
		
    </article>
	<!--Card 8-->
	<article class="card"></article>
  </main>
  <footer class="footer">
    <button class='submitt-button'>Submitt an article</button>
  </footer>
</body>
<script src='script.js'></script>
</html>
